src/main/asciidoc/_chapters/configuration.adoc

```diff
@@ -97,55 +97,76 @@ This section lists required services and some required system configuration.
 [[java]]
 .Java
 
-The following table summarizes the recommendation of the HBase community wrt deploying on various
-Java versions. A icon:check-circle[role="green"] symbol is meant to indicate a base level of
+The following table summarizes the recommendations of the HBase community with respect to deploying
+on various Java versions. The icon:check-circle[role="green"] symbol indicates a base level of
 testing and willingness to help diagnose and address issues you might run into. Similarly, an entry
-of icon:exclamation-circle[role="yellow"] or icon:times-circle[role="red"] generally means that
-should you run into an issue the community is likely to ask you to change the Java environment
-before proceeding to help. In some cases, specific guidance on limitations (e.g. whether compiling
-/ unit tests work, specific operational issues, etc) will also be noted.
+of icon:exclamation-circle[role="yellow"] means that, should you run into an issue, the community
+is likely to ask you to change the Java environment rather than proceeding to help on that Java
+version. This applies categorically to Non-LTS releases (see below). The
+icon:times-circle[role="red"] means that support has been dropped for this version or has been
+attempted and is known to not work. For combinations of newer JDK with older HBase releases, it's
+likely that there are known compatibility issues that cannot be addressed under our compatibility
+guarantees, making that combination impossible. In some cases, specific guidance on limitations
+(e.g. whether compiling / unit tests work, specific operational issues, etc) will also be noted.
+
+.Long-Term Support JDKs are Recommended
+[WARNING]
+====
+HBase recommends downstream users rely on JDK releases that are marked as Long-Term Supported
+(LTS), either from the OpenJDK project or vendors. At the time of this writing, the following JDK
+releases are NOT LTS releases and are NOT supported by the Apache HBase community: JDK9, JDK10,
```

Comment:
The word "supported" always turns my stomach (thanks, day-job) because people have such a wide breadth of what that means.

I liked your phrasing earlier around "If you encounter an issue with one of these JDK versions, our first ask will be to use a JDK version which has been tested". I'm not sure if there is a more concise way to encapsulate this.

(I'll also acknowledge that I'm overly sensitive to this terminology. If it's OK to others, then it's fine :) )

Reply:
Yeah you're right. "support" is not a great term, the the world of enterprise software. "You're unlikely to find receptive community engagement" is a bit long-winded, but honest. The earlier phrasing wasn't mine, but I'll try again with it as a model.